roles/passenger/templates/passenger.j2

```diff
@@ -11,7 +11,14 @@ server {
   set_real_ip_from 128.112.203.146;
   real_ip_header X-Real-IP;
 
-  access_log /var/log/nginx/access.log custom;
+  access_log /var/log/nginx/access.log custom if=$loadbalancer_enable;
```

Comment:
This variable name is a bit confusing - it sounds like we're ENabling something on the loadbalancers, when in fact, this would DISable log messages from the load balancers. How about a different var name?
```suggestion
  access_log /var/log/nginx/access.log custom if=$logging_enabled;
```

Reply:
the passenger would just stop writing the logs from the loadbalancers. It will record the upstream. I ran this branch on `catalog1.princeton.edu` for coherence check